docker/jupyter/Dockerfile

```diff
@@ -123,6 +123,10 @@ ENV USER jupyter-user
 ENV UID 1000
 ENV HOME /home/$USER
 
+# consistent with Welder docker definition here: https://github.com/DataBiosphere/welder/blob/master/project/Settings.scala
+ENV WELDER_USER welder-user
```

Comment:
This isn't really technically needed, but it makes notebooks in the Jupyter container display as:
```
-rw-rw-r-- 1 welder-user users 1157 Jun 28 01:43 np.ipynb
```
instead of 
```
-rw-r--r-- 1        1001 users 1157 Jun 28 01:50 Untitled.ipynb
```

Reply:
Nice, +1

Might be worth expanding this comment slightly to clarify that it's not strictly required.